vnext/Microsoft.ReactNative/TurboModulesProvider.cpp

```diff
@@ -7,6 +7,7 @@
 #include "pch.h"
 #include "TurboModulesProvider.h"
 #include <ReactCommon/TurboModuleUtils.h>
+#include "JsiApi.h"
 #include "JsiReader.h"
 #include "JsiWriter.h"
 #ifdef __APPLE__
```

Comment:
why do we have this clause? 🤔

Reply:
I am not sure. I think it was added by Zihan to adapt this code to RN for Mac. I guess we can revisit it later. It is out of scope of this PR.

---
In reply to: [550076891](https://github.com/microsoft/react-native-windows/pull/6804#discussion_r550076891) [](ancestors = 550076891)